using_images/db_images/mongodb.adoc

```diff
@@ -145,65 +147,125 @@ MongoDB settings can be configured with the following environment variables.
+and a
+link:../../architecture/core_concepts/pods_and_services.html#services[Service].
+
+The MongoDB templates should have been registered in the `openshift` project by
+the administrator during the
+link:../../admin_guide/install/first_steps.html[first steps] setup process.
+There are two templates available:
+
+* `mongodb-ephemeral` is for development/testing purposes only because it uses
+ephemeral storage for the database content.  This means that if the database
+Pod is restarted for any reason, such as the Pod being moved to another node
+or the DeploymentConfig being updated and triggered a redeploy, all data will
+be lost.
+* `mongodb-persistent` uses a persistent volume store for the database data
+which means the data will survive a Pod restart.  Using persistent volumes
+requires a persistent volume pool be defined in the OpenShift deployment.
+Instructions for setting up the pool are located
+link:../../admin_guide/persistent_storage_nfs.html[here].
+
+
+You can find instructions for instantiating templates by following these
+link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
+
+Once you have instantiated the service, you can copy the username, password, and
+database name environment variables into a DeploymentConfig for another
+component that intends to access the database.  That component can then access
+the database via the Service that was defined.
+
+[[using-mongodb-replication]]
+== Using MongoDB Replication
+
+ifdef::openshift-origin[]
+[IMPORTANT]
+====
+Replication support provided by the MongoDB image is experimental and should not
+be used in production.
+====
+endif::[]
 
-You can find instructions for instantiating templates by following these link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+Enabling clustering for database images is currently in
+link:../../whats_new/ose_3_0_release_notes.html#technology-preview[Technology
+Preview] and not intended for production use.
+====
+endif::[]
 
-Once you have instantiated the service, you can copy the username, password, and database name environment variables into a DeploymentConfig for another component that intends to access the database.  That component can then access the database via the Service that was defined.
+Red Hat provides a proof-of-concept
+link:../../architecture/core_concepts/templates.html[template] for MongoDB
+replication; you can obtain the
+https://github.com/openshift/mongodb/tree/master/2.4/examples/replica[example
+template from GitHub].
 
-== Using MongoDB replication
+For example, to upload the example template into the current project's template
+library:
 
-IMPORTANT: The replication support provided by the MongoDB image is experimental and
-should not be used in production.
+====
+----
+$ oc create -f \
+    https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb_controller.json
```

Comment:
@adellape so it is `https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json`